pkg/master/master_test.go

```diff
@@ -30,7 +30,7 @@ func TestGetServersToValidate(t *testing.T) {
 	config := Config{}
 	fakeClient := tools.NewFakeEtcdClient(t)
 	fakeClient.Machines = []string{"http://machine1:4001", "http://machine2", "http://machine3:4003"}
-	config.EtcdHelper = tools.EtcdHelper{fakeClient, latest.Codec, nil}
+	config.EtcdHelper = tools.EtcdHelper{fakeClient, latest.Codec, nil, ""}
```

Comment:
Use testetcd.PathPrefix() instead of ""; also s/tools.PathPrefix/testetcd.PathPrefix/. Thanks.